### PR TITLE
Start removing references to conduit in the web UI

### PR DESCRIPTION
**Pull request by rmars**
This PR starts removing all references to the word "Conduit" in the web UI. 
In the interest of not making huge changes all at once, I'll gradually start moving away 
from the usage of "conduit" in the Web UI. For example, there are a lot of components that 
have conduit in their names but they don't need to.

This branch is mostly component / variable names. There should be no visible changes except the spinner is no longer a Conduit spinner, so this can be merged whenever.
See #1262 for visible branding changes.

- Rename ConduitLink to PrefixedLink
- Remove ConduitSpinner in favour of antd.Spin
- Remove css classnames that are conduit- centered
- Parameterize the current Product Name so that it's easier to change in the future

Tracking ticket: https://github.com/linkerd/linkerd/issues/2018

### CODE REVIEW

File: web/app/css/styles.css [hunk 1]
```diff
@@ -74,6 +74,14 @@ h2, h3, h4, h5, h6 {
   float: right;
 }
 
+/* center the spinner */
```
Comment (siggy) [inline web/app/css/styles.css]: not sure if this behavior already existed, spinner appears centered over page content, but not window:
screen shot 2018-07-03 at 4 12 00 pm

File: web/app/js/components/Namespace.jsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ class Namespaces extends React.Component {
     return (
       <div className="page-content">
         { !this.state.error ? null : <ErrorBanner message={this.state.error} /> }
-        { !this.state.loaded ? <ConduitSpinner /> : (
+        { !this.state.loaded ? <Spin size="large" /> : (
```
Comment (siggy) [inline web/app/js/components/Namespace.jsx]: if there's something larger than `large`, i'd say go for it.

File: web/app/css/styles.css (hunk 1 above)
Comment (rmars) [reply web/app/css/styles.css]: yeah, this was the existing behaviour!

File: web/app/js/components/Namespace.jsx (hunk 2 above)
Comment (rmars) [reply web/app/js/components/Namespace.jsx]: haha, no large is the biggest it gets!
https://ant.design/components/button/
This is already bigger than the existing conduit spinner though!

File: web/app/js/components/Version.jsx [hunk 3]
```diff
@@ -46,11 +49,11 @@ class Version extends React.Component {
...
-        Running Conduit {this.props.releaseVersion}<br />
+        Running {this.props.productName || "controller"} {this.props.releaseVersion}<br />
         {this.renderVersionCheck()}
       </div>
     );
   }
 }
 
-export default Version;
+export default withContext(Version);
```
Comment (grampelberg) [inline web/app/js/components/Version.jsx]: Aw yeah! That was easy =)

**Pull request merged**